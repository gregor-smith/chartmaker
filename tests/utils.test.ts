describe('readFileText', () => {
    test.todo('reads file containing utf-8 text')


    test.todo('errors for file containing invalid text')
})


describe('findIndex', () => {
    test.todo('returns index when item matching predicate is in array')


    test.todo('returns null when no item in array matches predicate')
})


describe('elementToDataURI', () => {
    test.todo('returns data uri for scaled image')
})


describe('jsonToDataURI', () => {
    test.todo('returns escaped JSON data uri')
})
