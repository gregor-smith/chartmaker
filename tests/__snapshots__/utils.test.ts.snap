// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`jsonToDataURI returns escaped JSON data uri 1`] = `"data:application/json;charset=utf-8,%7B%22test%22%3A%22test%20json%22%7D"`;
